keyman_trust_linux.go

```diff
@@ -4,71 +4,90 @@ import (
 	"fmt"
 	"os"
 	"os/exec"
-	"os/user"
+	"path/filepath"
 )
 
-func DeleteTrustedRootByName(commonName string, prompt string) error {
-	nssdb, err := getUserNssdb()
-	if err != nil {
-		return err
+var (
+	nssDBs = []string{
+		filepath.Join(os.Getenv("HOME"), ".pki/nssdb"),
+		filepath.Join(os.Getenv("HOME"), "snap/chromium/current/.pki/nssdb"), // Snapcraft
+		"/etc/pki/nssdb", // CentOS 7
 	}
+	FirefoxProfile = os.Getenv("HOME") + "/.mozilla/firefox/*"
+)
 
-	cmd := exec.Command("certutil", "-d", nssdb, "-D", "-n", commonName)
-	out, err := cmd.CombinedOutput()
-	if err != nil {
-		return fmt.Errorf("Unable to run certutil command: %s\n%s", err, out)
-	}
-	return nil
+func DeleteTrustedRootByName(commonName string, prompt string) error {
+	return forEachNSSProfile(func(profile string) error {
+		cmd := exec.Command("certutil", "-d", profile, "-D", "-n", commonName)
+		out, err := cmd.CombinedOutput()
+		if err != nil {
+			return fmt.Errorf("Unable to run certutil command: %s\n%s", err, out)
```

Comment:
Using '%w' instead of '%s' will capture the error for later unwrapping if needed.